packages/apps/patterns/react-ui/src/components/InvitationList/InvitationList.tsx

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2023 DXOS.org
+//
+import { Root as AccordionRoot } from '@radix-ui/react-accordion';
+import React from 'react';
+
+import type { CancellableInvitationObservable } from '@dxos/client';
+
+import { InvitationListItem, InvitationListItemProps } from './InvitationListItem';
+
+export interface InvitationListProps extends Omit<InvitationListItemProps, 'invitation' | 'value'> {
+  invitations: CancellableInvitationObservable[];
+}
+
+export const InvitationList = ({ invitations, ...invitationProps }: InvitationListProps) => {
+  return (
+    <AccordionRoot type='single' collapsible>
+      {invitations.map((invitation, index) => {
+        const value = invitation.invitation?.invitationId ?? `inv_${index}`;
```

Comment:
Yes, this is an unusual case but all of the view states here need to render in a way that occupies the usual amount of space so the layout gets the right inherent height, ultimately so that the modal isn’t changing size between states.